modules/claims_api/app/swagger/claims_api/v2/swagger.json

```diff
@@ -937,6 +937,7 @@
       }
     }
   },
+  "basePath": "/services/benefits/v2",
```

Comment:
I think if you rebase with Master and run `rake rswag:claims_api:run` again, it'll handle getting rid of this `basePath` property for you.  It's something that Dan added (merged in today) that helps keeps us up-to-standards with OAS3.